react/typings/vtex.native-types.d.ts

```diff
@@ -1,10 +1,18 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 declare module 'vtex.native-types' {
   import { ComponentType } from 'react'
+  import { MessageDescriptor } from 'react-intl'
 
   export const IOMessage: ComponentType<{
     id: string
     values: Record<string, any>
     ['data-testid']: string
   }>
+
+  export const formatIOMessage: (
```

Comment:
Why not use the exported types from the app?

Reply:
I'll try it out, but I think they're not quite right... If that's the case, I'll open a PR to fix types for `vtex.native-types` and come back to this :)

Reply:
Yep, the types are not quite there. I'll open a PR to fix them

Reply:
There you go: https://github.com/vtex-apps/native-types/pull/31